packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -142,6 +142,11 @@ export const configSchema = {
     _default: '',
     _description: 'Custom label for patient chart button',
   },
+  useActiveVisitsTable: {
+    _type: Type.Boolean,
+    _default: false,
+    _description: 'If true, renders the ActiveVisitsTable instead of the default queue table',
+  },
```

Comment:
As discussed, the goal here is not to support 2 different table implementations, but to refactor the table to use the new component design so only one table is maintained, so this should not be a configuration option.